File: sample-code/fibonacci.zig
const std = @import("std");

fn fib(n: u32) u32 {
    return if (n < 2) n else fib(n - 1) + fib(n - 2);
}

pub fn main() void {
    const n: u32 = 10;
    std.debug.print("{}\n", .{fib(n)});
}

File: sample-code/fibonacci.pl
# Fibonacci (Perl)

use strict;
use warnings;

sub fib {
  my ($n) = @_;
  return $n if $n < 2;
  return fib($n - 1) + fib($n - 2);
}

for my $i (0..11) {
  print "$i ", fib($i), "\n";
}
